src/app/component/quiz/quiz.component.scss

```diff
@@ -1,59 +1,105 @@
+.quiz_card {
+  height: auto;
+}
 
-.card{
-    margin: 0 auto; 
-    margin-bottom: 50px;
-    border-radius: 15px;
-    height: auto;
-    
-}
-.card-body{
+.header_txt {
+  position: relative;
+  z-index: 2;
+  height: 70px;
+  padding: 0 15px;
+  display: flex;
```

Comment:
manage it with boostrap class no need to write css for it 
d-flex align-items-center justify-content-space-between

Reply:
ok, i removed css and manage it with boostrap class